.github/workflows/frontend.yml

```diff
@@ -45,45 +46,32 @@ jobs:
           app_id: ${{ secrets.SENTRY_INTERNAL_APP_ID }}
           private_key: ${{ secrets.SENTRY_INTERNAL_APP_PRIVATE_KEY }}
 
-      - name: Check for frontend file changes
-        uses: getsentry/paths-filter@master
-        id: changes
-        with:
-          list-files: shell
-          token: ${{ github.token }}
-          filters: .github/file-filters.yml
```

Comment:
I'm removing this since it's not being used. Check that we don't use `steps.changes`  but we use `needs.files-changed`.